src/service/logService.js

```
@@ -0,0 +1,14 @@
+import axiosInstance from "./axiosInstance";
+
+class LogService {
+    static async getLogs(){
+        try {
+            const response = await axiosInstance.get(`/logs`);
+            return response.data;
+        }catch(error){
+            throw error;
+        }
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)